packages/client/e2e/_utils/page.ts

```diff
@@ -39,13 +29,11 @@ export const GEOLOCATION_POSITIONS = {
 
 type SetupAndGetUserContextArgs = {
     browser: Browser;
-    userIndex: number;
     knownSearches: KnownSearchesRecord;
     town?: keyof typeof GEOLOCATION_POSITIONS;
 };
 export async function setupAndGetUserPage({
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❔ question:** ‏Maybe we would like to rename this function so that we immediately know that it creates a user? Like `setupPageAndSignUpUser`?

When I read the e2e tests below, it did not seem evident to me that `setupAndGetUserPage` was actually creating a new user.